adr/adr-guidelines.md

```diff
@@ -0,0 +1,26 @@
+# Architectural Decision Record Formatting Checklist
+Below is a formatting checklist with some guidelines that serves to help keep our ADRs looking clean and uniform. 
+
+1. See [adr/001-architecture-decision-records.md](adr/001-architecture-decision-records.md) for template and section guidelines/requirements.
+
+
+2. Review ‘Context’ section for grammar, typos, and clarity. ** **Keep in mind -** ** readers may have less inside knowledge or experience with the project when reading these documents.
+
+
+2. **Bulleted Lists:** Full sentences get periods and first letter capitalized. Fragmented sentences have no period and aren’t capitalized.
+
+
+3. **Bulleted Lists:** Embolden bulleted titles with colons
+
+
+4. Make sure line spacing is appropriate give a full line of space between bullet points
+
+
+5. **DO NOT** use the word ‘trusted’ in reference to the Intermediary Application
```

Comment:
@halprin are we supposed to not-talk about not talking about 'trusted', or are we just supposed to not talk about it?

Reply:
lololol.  I'm OK removing this #5 item.